content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -0,0 +1,346 @@
+---
+reviewers:
+- hasheddan
+- pjbgf
+- saschagrunert
+title: Seccomp
+content_type: tutorial
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+
+Seccomp stands for secure computing mode and has been a feature of the Linux
+kernel since version 2.6.12.  It can be used to sandbox the privileges of a
+process, restricting the calls it is able to make from userspace into the
+kernel. Kubernetes lets you automatically apply seccomp profiles loaded onto a
+Node to your Pods and containers.
+
+Identifying the privileges required for your workloads can be difficult. In this
+tutorial, we will go through how to load seccomp profiles into a local
+Kubernetes cluster, how to apply them to a Pod, and how you can begin to craft
+profiles that give only the necessary privileges to your container processes.
+
+## {{% heading "objectives" %}}
+
+* Learn how to load seccomp profiles on a node
+* Learn how to apply a seccomp profile to a container
+* Observe auditing of syscalls made by a container process
+* Observe behavior when a missing profile is specified
+* Observe a violation of a seccomp profile
+* Learn how to create fine-grained seccomp profiles
+* Learn how to apply a container runtime default seccomp profile
+
+## {{% heading "prerequisites" %}}
+
+In order to complete all steps in this tutorial, you must install
+[kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and
+[kubectl](../../tasks/tools/install-kubectl). We will show examples with both
+`alpha` (pre-v1.19) and GA seccomp functionality, so make sure that your cluster
```

Comment:
alpha (pre-v1.19) and generally available seccomp functionality, so make sure that your cluster

Reply:
@hasheddan , Why is it important to compare the `alpha` seccomp with `GA` seccomp?
It makes the pathway through the tutorial more complicated.

Reply:
@kbhawkey If someone is using a Kubernetes version pre-v1.19 the GA behavior will not be available. Because it takes time for people to update to newer versions of Kubernetes I feel like it makes sense to demonstrate how seccomp can be used in both settings.

Reply:
@hasheddan you might not know that Kubernetes documentation is also available for previous versions.

In the documentation for v1.19 and later you can tell readers to look back at the documentation for previous releases - eg https://v1-18.docs.kubernetes.io/ (and this can be different text).
That's how I'd do it.

If you want to achieve this we can explain how.

Reply:
@sftim I have implemented the tabs approach as suggested in https://github.com/kubernetes/website/pull/21278#discussion_r454694898. This feels like a clean experience to me, but I am happy to remove the alpha sections and make a separate revision of this tutorial for 1.18 if needed. Let me know what you think